cmd/notary/tuf.go

```diff
@@ -834,3 +839,30 @@ func (a *authRoundTripper) RoundTrip(req *http.Request) (*http.Response, error)
 	// Return the last response
 	return resp, nil
 }
+
+func publish(cmd *cobra.Command, doPublish bool, gun string, config *viper.Viper, passRetriever notary.PassRetriever) error {
```

Comment:
also, I think it'd be nice to rename this to `maybePublish` or `maybeAutoPublish` so it's clear that we may not actually publish